src/main/java/com/pet/bankservice/util/InitialDataInject.java

```diff
@@ -0,0 +1,41 @@
+package com.pet.bankservice.util;
+
+import com.pet.bankservice.entity.Role;
+import com.pet.bankservice.entity.User;
+import com.pet.bankservice.service.RoleService;
+import com.pet.bankservice.service.UserService;
+import javax.annotation.PostConstruct;
+import java.time.LocalDate;
+import java.util.HashSet;
+import java.util.Set;
+import org.springframework.stereotype.Component;
+
+@Component
+public class InitialDataInject {
+    private final UserService userService;
+    private final RoleService roleService;
+
+    public InitialDataInject(UserService userService, RoleService roleService) {
```

Comment:
I am not sure that you should push that code. Anyway be consistent and use `@AllArgsConstructor` here as well.

Reply:
done